Sources/BraintreeCore/Authorization/ClientAuthorization.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+/// :nodoc: Form of authorization to interact with the Braintree gateway.
+@_documentation(visibility: private)
+public protocol ClientAuthorization {
```

Comment:
Yeh, and it has to be public since `BTClientToken` is public. DropIn is using `BTClientToken` directly. I know v6 isn't compatible with DropIn, but I didn't want to mess with that in this PR